radio/src/targets/nv14/board.h

```diff
@@ -140,6 +144,13 @@ void EXTERNAL_MODULE_OFF();
 #define BLUETOOTH_MODULE_ON()           GPIO_ResetBits(BLUETOOTH_ON_GPIO, BLUETOOTH_ON_GPIO_PIN)
 #define BLUETOOTH_MODULE_OFF()          GPIO_SetBits(BLUETOOTH_ON_GPIO, BLUETOOTH_ON_GPIO_PIN)
 
+#if defined(BOOT)
+
+#define USB_SW_TO_INTERNAL_MODULE() GPIO_SetBits(USB_SW_GPOIO, USB_SW_PIN);
+#define USB_SW_TO_MCU() GPIO_ResetBits(USB_SW_GPOIO, USB_SW_PIN);
```

Comment:
I think these 2 could be solely in the `.cpp` code. No need to export anything here.

Reply:
done